src/redux/conference-details.js

```diff
@@ -0,0 +1,80 @@
+var keyMirror = require('keymirror');
+var api = require('../mixins/api.jsx').api;
+
+var Types = keyMirror({
+    SET_DETAILS: null,
+    SET_DETAILS_FETCHING: null,
+    SET_DETAILS_ERROR: null
+});
+
+module.exports.detailsReducer = function (state, action) {
+    if (typeof state === 'undefined') {
+        state = {};
+    }
+    switch (action.type) {
+    case Types.SET_DETAILS:
+        return action.details;
+    case Types.SET_DETAILS_FETCHING:
+        return {fetching: action.fetching};
+    case Types.SET_DETAILS_ERROR:
+        return {error: action.error};
+    default:
+        return state;
+    }
+};
+
+module.exports.setDetailsError = function (error) {
+    return {
+        type: Types.SET_DETAILS_ERROR,
+        error: error
+    };
+};
+
+module.exports.setDetails = function (details) {
+    return {
+        type: Types.SET_DETAILS,
+        details: details
+    };
+};
+
+module.exports.setDetailsFetching = function () {
+    return {
+        type: Types.SET_DETAILS_FETCHING,
+        fetching: true
+    };
+};
+
+module.exports.startGetDetails = function (id) {
+    return function (dispatch) {
+        dispatch(module.exports.setDetailsFetching());
+        dispatch(module.exports.getDetails(id));
+    };
+};
+
+module.exports.getDetails = function (id) {
+    return function (dispatch) {
+        api({
+            uri: '/conference/' + id + '/details'
+        }, function (err, body) {
+            if (err) {
+                dispatch(module.exports.setDetailsError(err));
+                return;
+            }
+
+            if (typeof body !== 'undefined') {
```

Comment:
I put it at the end already since it seemed like it might be easier (so there's no additional conditional check for if it's `err` or not inside that `or`). That cool with you?